proto/metrics.rs

```diff
@@ -2711,7 +2711,7 @@ impl ::protobuf::ProtobufEnum for MetricType {
     }
 
     fn values() -> &'static [Self] {
-        static values: &'static [MetricType] = &[
```

Comment:
can the rust version used in TiKV support removing static?

Reply:
Yes.
```
neil:rust-prometheus/ (ov/clippy) $ rustc -V
rustc 1.21.0-nightly (215e0b10e 2017-08-08)
neil:rust-prometheus/ (ov/clippy) $ cargo check
    Finished dev [unoptimized + debuginfo] target(s) in 0.0 secs
```